src/lib.rs

```diff
@@ -91,6 +103,21 @@ pub mod module {
 
         /// A user's role is unregistered.
         RoleUnregistered { user: T::AccountId },
+
+        /// A manager role mint some fund and deposit to an account.
```

Comment:
Should write these in the past tense.
"A manager role has minted some funds into an account"

Same as other comments for Events